.github/workflows/first-interaction.yml

```diff
@@ -22,10 +22,17 @@ jobs:
 
           We are glad you are here and appreciate your contribution. Please keep in mind our [community participation guidelines](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CODE_OF_CONDUCT.md). 📜
           Also, please check [existing open issues](https://github.com/InsightSoftwareConsortium/ITK/issues) and consider discussion on the [ITK Discourse](https://discourse.itk.org). 📖
+
+          This is an automatic message. Hopefully, this is not all that will happen ! But allow for some time to the ITK
+          community to be able to read the issue and comment on it.
```

Comment:
An adjustment that could be more succinct and positive.

> Hopefully, this is not all that will happen ! But allow for some time to the ITK￼community to be able to read the issue and comment on it.

to

> Allow for time for the ITK community to be able to read the issue and comment on it.

Reply:
Done in [b774fd5](https://github.com/InsightSoftwareConsortium/ITK/pull/4133/commits/b774fd5358112c019466b95db72f87f97b2f6671).